server/src/main/java/org/elasticsearch/action/search/RestOpenPointInTimeAction.java

```diff
@@ -47,6 +47,7 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
         openRequest.routing(request.param("routing"));
         openRequest.preference(request.param("preference"));
         openRequest.keepAlive(TimeValue.parseTimeValue(request.param("keep_alive"), null, "keep_alive"));
+        openRequest.allowPartialSearchResults(request.paramAsBoolean("allow_partial_search_results", false));
```

Comment:
Naming of the param is up for discussion.. Maybe we could rename it to `allow_missing_shards` or something more specific? 🤔

Reply:
I prefer uniformity with the `_search` parameter.